obspy/io/sac/core.py

```diff
@@ -377,6 +377,11 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :param fsize: Check if file size is consistent with theoretical size
         from header. Defaults to ``True``.
     :type fsize: bool
+    :param byteorder: If omitted or None, automatic byte-order checking is
+        done, starting with native order. If byteorder is specified and
+        incorrect, a :class:`SacIOError` is raised. Only valid for binary
+        files.
+    :type byteorder: str {'little', 'big'}, optional
```

Comment:
Btw, we have `<` and `>` in most (all other?) parts of the codebase.. maybe these should be accepted as aliases here?

On an even wider scope, maybe all across obspy `little`/`<` and `big`/`>` should be accepted interchangeably, but this is probably something for a separate PR and against master then.

N.B.: There is some related stuff in `io/sac/util.py` in `is_same_byteorder` and also some stuff in `io/sac/core.py` where even `0` and `1` are treated as byteorder indicators (also in some mseed code). But like I said, maybe postpone for different PR.

Reply:
I'm in support of this, and I'm happy to start an empty PR so it doesn't get lost.